### ISSUE TITLE
Build currently broken

### ISSUE THREAD

**Issue by Jeffrey-P-McAteer** [Jeffrey-P-McAteer/UMW_ACM_Programming_Challenge_Proctor#5]
The sandbox container fails to build, something about java being dumb:
```
Step 11 : RUN set -x   && apk add --no-cache      openjdk8="$JAVA_ALPINE_VERSION"   && [ "$JAVA_HOME" = "$(docker-java-home)" ]
 ---> Running in 7c61ce570714
+ apk add --no-cache openjdk8=8.92.14-r1
fetch http://dl-cdn.alpinelinux.org/alpine/v3.4/main/x86_64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/v3.4/community/x86_64/APKINDEX.tar.gz
ERROR: unsatisfiable constraints:
  openjdk8-8.111.14-r0:
    breaks: world[openjdk8=8.92.14-r1]
The command '/bin/sh -c set -x   && apk add --no-cache      openjdk8="$JAVA_ALPINE_VERSION"   && [ "$JAVA_HOME" = "$(docker-java-home)" ]' returned a non-zero code: 1
```
Will look at sometime tomorrow.

**Comment by Jeffrey-P-McAteer** [comment]
May have something to do with java being updated and our variables for versions are out of date (Sandbox/Dockerfile:29 or so). I'll have to think of ideas to automate updating this.

**Comment by Jeffrey-P-McAteer** [comment]
Fixed with 3c98862
